### PR TITLE
Updated logging.properties, build.xml, JUnit 4.11

### PR DESCRIPTION
**Pull request by BrettMichaelJohnson**

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -129,7 +127,7 @@ lib/plexi submodule or add the the command line argument
            includeantruntime="false" encoding="utf-8"
            source="${compile.java.version}" target="${compile.java.version}">
       <bootclasspath path="${compile.java.bootclasspath}"/>
-      <compilerarg line="-Xlint -Xlint:-serial -Xlint:-path"/>
+      <compilerarg line="-Xlint -Xlint:-serial"/>
```
Comment (wiarlawd) [inline build.xml]: Jace.jar generates path warnings. Was this change just copied from elsewhere?

File: build.xml [hunk 2]
```diff
@@ -193,7 +200,7 @@ lib/plexi submodule or add the the command line argument
         <attribute name="Class-Path" value="${tmp.adaptor.classpath}"/>
         <section name="com/google/enterprise/adaptor/filenet/">
           <attribute name="Implementation-Title"
-            value="Google FileNet Adaptor"/>
+            value="Google Search Appliance Connector for FileNet"/>
```
Comment (wiarlawd) [inline build.xml]: I had also changed the descriptions at the top of build.xml, to a plain "Connector for Foo".

File: build.xml (hunk 1 above)
Comment (BrettMichaelJohnson) [inline build.xml]: Documentum

Comment (BrettMichaelJohnson) [inline build.xml]: Just matched Documentum build.xml. Restored -Xlint:-path

File: build.xml (hunk 2 above)
Comment (BrettMichaelJohnson) [inline build.xml]: Done.

**Pull request merged**